### PR TITLE
docs: move accessibility document over

### LINKED ISSUE #89: Move accessibility docs to design
Have this as their own section

### PR DESCRIPTION
**Pull request by K-Beicher**
## Description

- This PR moves the accessibility why document over to design, and re-words a bit of it.
- These changes are done because the topic fits better under its own heading in design and not as a traditional why decision post.

## Related Issues

Closes #89 

## Reviewer Focus

This PR only needs a quick review.

Mostly the same text as before, but the hidden sections have been removed.

## Checklist

For general documentation:

- [x] Spell-check
    - [ ] US
    - [ ] UK
- [ ] Did the page(s) preview correctly on your machine without breaking
- [ ] New category words (keywords) (if any) added to the code snippet file

### CODE REVIEW

File: accessibility/index.qmd [hunk 1]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+## Introduction
+
+It is important to us that the outputs of the Seedcase Project
+(including the websites, the software products, and the teaching
+material) are as accessible as possible. This includes people with
+disabilities, people who are older, people who are not native English
+speakers, and people who are not familiar with technology. The Seedcase
+project is no exception. We want to make all output of the Seedcase
+Project as accessible as possible for all potential users. This includes
+both the websites, the software products, and the teaching material.
```
Comment (signekb) [inline accessibility/index.qmd:16]:
```suggestion
speakers, and people who are not familiar with technology.
```
There seems to be some duplicated text here

File: accessibility/index.qmd [hunk 2]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+
+## Decision drivers
+
```
Comment (signekb) [inline accessibility/index.qmd:19]:
```suggestion
```
This header seems to be a leftover from when this was a decision post. The next section could fit under the Introduction header above.

File: accessibility/index.qmd [hunk 3]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+We want to ensure that a multitude of users will be able to work with
+our software and interact with our documentation. At first this was a
+question of ensuring that our colour scheme was accessible to people
+with colour blindness, which in turn lead us to consider other
+accessibility issues. We want to ensure that the Seedcase Project is as
+inclusive as possible, so it's accessible to as many users as possible.
+This is also a requirement for the Seedcase Project to be successful in
+the long term.
+
+## Considered options
```
Comment (signekb) [inline accessibility/index.qmd:29]:
```suggestion
```
Same with this header

File: accessibility/index.qmd [hunk 4]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+
+There are a number of ways of looking at accessibility, the most common
+are accessible content and visual accessibility. We will work with both.
+
+### Accessible content
```
Comment (signekb) [inline accessibility/index.qmd:34]:
```suggestion
## Accessible content
```
Then this header can be level 2 instead

File: accessibility/index.qmd [hunk 5]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+-   are the images described in a way that is compatible with screen
+    readers (i.e., do we include alt texts)?
+-   are the buttons and links named in a way that is easy to understand?
+-   are the interactive elements easy to navigate with a keyboard?
+-   are any landmarks/headings coded in a way that makes guides a screen
+    reader in a logical way?
+-   if we end up making videos or audio content, is there a transcript
+    available?
+
+### Visual accessibility
```
Comment (signekb) [inline accessibility/index.qmd:55]:
```suggestion
## Visual accessibility
```

File: accessibility/index.qmd [hunk 6]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+## Considered options
+
+There are a number of ways of looking at accessibility, the most common
+are accessible content and visual accessibility. We will work with both.
+
+### Accessible content
+
+Accessible content is content that is easy to understand and use for
+everyone. Things like accessible naming, and using a polite setting for
+the hidden text. Ensuring that the language isn't too technical or
```
Comment (signekb) [inline accessibility/index.qmd:38]:
```suggestion
everyone. This includes things like accessible naming and using a polite
setting for hidden text. Ensuring that the language isn't too technical or
```

File: accessibility/index.qmd [hunk 7]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+complex, and that pictures and potential audio/visual content is
```
Comment (signekb) [inline accessibility/index.qmd:39]:
```suggestion
complex, and that pictures and potential audio/visual content are
```

File: accessibility/index.qmd [hunk 8]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+Things to check for during development:
+
+-   is the text to complex?
+-   is the language too technical?
+-   are the images described in a way that is compatible with screen
+    readers (i.e., do we include alt texts)?
+-   are the buttons and links named in a way that is easy to understand?
+-   are the interactive elements easy to navigate with a keyboard?
+-   are any landmarks/headings coded in a way that makes guides a screen
+    reader in a logical way?
```
Comment (signekb) [inline accessibility/index.qmd:51]:
```suggestion
    reader in a logical way? (i.e., do we avoid skipping header levels?)
```
or something like this?

File: accessibility/index.qmd [hunk 9]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+-   is the colour scheme accessible to people with colour blindness?
+-   is the colour ratio between text and background high enough (minimum
+    4.5:1)?
+-   is the text easy to read (large enough font)?
+-   is the line length suitable (between 45 and 75 char)?
+-   are the meaning the icons convey established (like a pencil for
+    edit, or a square for stop)?
+-   are the buttons easy to click, with a tap-target a minimum of 44x44
+    px and not too closely spaced?
+-   are the interactive states easy to see (default, focus/hover,
```
Comment (signekb) [inline accessibility/index.qmd:75]:
```suggestion
-   are the interactive states of buttons easy to see (default, focus/hover,
```
Or do these states include other elements than buttons?

File: accessibility/index.qmd [hunk 10]
```diff
@@ -3,3 +3,95 @@ title: "Accessibility"
...
+    active/pressed, deactivated)?
+
+## Decision outcome
```
Comment (signekb) [inline accessibility/index.qmd:78]:
```suggestion
## Conclusion
```
Or at least something different than "Decision outcome"? :)

**Review by signekb — CHANGES_REQUESTED** [review]
Very nice! I have some suggestions now that it's not a traditional decision post anymore

**Review by signekb — DISMISSED** [review]

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**